policyengine_us/parameters/gov/states/ga/tax/income/credits/low_income_credit/amount.yaml

```diff
@@ -0,0 +1,34 @@
+description: Georgia provides the following low income tax credit amount based on their agi.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label: Georgia low income tax credit amount
+  reference:
+    - title: georgia low income credit wooksheet(page22)
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=0d6fd3c6-6e65-4928-9679-0b33bed98769&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3SK-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr0&pditab=allpods&ecomp=7s65kkk&earg=sr0&prid=8be67d32-0de2-4977-a057-d9d01bc94c88
```

Comment:
add tax form for 2022 and 2021 tax forms - should be 3 references total (legal, 2022 form, 2021 form)